waku/waku_rln_relay/rln_relay.nim

```diff
@@ -33,16 +34,19 @@ import
 logScope:
   topics = "waku rln_relay"
 
-type WakuRlnConfig* = object
-  rlnRelayDynamic*: bool
-  rlnRelayCredIndex*: Option[uint]
-  rlnRelayEthContractAddress*: string
-  rlnRelayEthClientAddress*: string
-  rlnRelayCredPath*: string
-  rlnRelayCredPassword*: string
-  rlnRelayTreePath*: string
-  when defined(rln_v2):
-    rlnRelayUserMessageLimit*: uint64
+type 
+  WakuRlnConfig* = object
+    rlnRelayDynamic*: bool
+    rlnRelayCredIndex*: Option[uint]
+    rlnRelayEthContractAddress*: string
+    rlnRelayEthClientAddress*: string
+    rlnRelayCredPath*: string
+    rlnRelayCredPassword*: string
+    rlnRelayTreePath*: string
+    # defined in waku_archive, but do not want to import it here
```

Comment:
Sorry I can't quite understand the comment:)

Reply:
addressed in efd1b017